packages/cli/src/eventbus/message-event-bus-destination/message-event-bus-destination-webhook.ee.ts

```diff
@@ -95,9 +100,56 @@ export class MessageEventBusDestinationWebhook
 		if (options.sendPayload) this.sendPayload = options.sendPayload;
 		if (options.options) this.options = options.options;
 
+		const axiosSetting = this.buildAxiosSetting(options);
+
+		this.axiosInstance = axios.create(axiosSetting);
+
 		this.logger.debug(`MessageEventBusDestinationWebhook with id ${this.getId()} initialized`);
 	}
 
+	private buildAxiosSetting(
+		options: MessageEventBusDestinationWebhookOptions,
+	): CreateAxiosDefaults {
+		const axiosSetting: CreateAxiosDefaults = {
+			headers: {},
+			method: options.method as Method,
+			url: options.url,
+			maxRedirects: 0,
+		} as AxiosRequestConfig;
+
+		if (options.options?.redirect?.followRedirects) {
+			axiosSetting.maxRedirects = this.options.redirect?.maxRedirects;
+		}
+
+		axiosSetting.proxy = options.options?.proxy;
+
+		axiosSetting.timeout = options.options?.timeout ?? 10 * Time.seconds.toMilliseconds;
```

Comment:
Is a 10-second timeout a bit generous? Maybe 5 seconds? Or am I being too optimistic?!